dockerbuilder/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:14.04
```

Comment:
Up until the 22nd it was the most recent LTS, and I understand that 15.x has having a bad rep for stability.